docs/external_system.md

```diff
@@ -366,25 +351,25 @@ SAT uses S3 storage for several purposes, most importantly to store the site-spe
 1. Ensure the files are readable only by `root`.
 
     ```screen
-    ncn-m001# touch /root/.config/sat/s3_access_key \
+    $ touch /root/.config/sat/s3_access_key \
         /root/.config/sat/s3_secret_key
```

Comment:
These commands need to use the user's home directory. So this would work:

```suggestion
    $ touch ~/.config/sat/s3_access_key \
        ~/.config/sat/s3_secret_key
```

Reply:
Updated in next commit